steps/preprocessing.py

```diff
@@ -42,13 +50,15 @@ def save(self, filepath):
 
 class TextCleaner(BaseTransformer):
     def __init__(self, drop_punctuation, drop_newline, drop_multispaces,
-                 all_lower_case, fill_na_with, deduplication_threshold):
+                 all_lower_case, fill_na_with, deduplication_threshold, anonymize, apostophes):
```

Comment:
Both `anonymize`, and `apostophes` are boolean, so it might be beneficial for the broader community to move it as a parameter to the `config.yaml`, to easily control this behavior.

If not it should be handled in the: [pipeline_config.py:L19](https://github.com/neptune-ml/kaggle-toxic-starter/blob/master/pipeline_config.py#L19)

Reply:
Implemented in `pipeline_config.py:L19` as of now

Reply:
@shaz13 thanks, I can see, that you added two more params to `neptune.yaml` -> :+1:

Reply:
:+1: